files/templates/mbta/login/messages/messages_es.properties

```diff
@@ -3,8 +3,8 @@ doRegister=Crea tu cuenta MBTA
 noAccount=¿Nueva usuaria?
 doForgotPassword=¿Has olvidado tu contraseña?
 loginAccountTitle=Inicie sesión en su cuenta MBTA
-backToLoginRegistration=«Inicia sesión con tu cuenta actual
-backToLoginResetPassword=«Volver al inicio de sesión
+backToLoginRegistration=« Inicia sesión con tu cuenta actual
```

Comment:
is this intended? (space between the quotes and beginning of line)

Reply:
Yes, this is on a button and the `«` is used as an arrow to go back. The machine translation made those actual quote characters in most languages, so I had to fix that back.